Tools/WebKitTestRunner/TestController.cpp

```diff
@@ -1684,6 +1703,7 @@ bool TestController::runTest(const char* inputLine)
 
     m_currentInvocation->invoke();
     m_currentInvocation = nullptr;
+    m_mainResourceURL = 0;
```

Comment:
We prefer `nullptr` in these cases where you're clearing out a smart pointer.

```suggestion
    m_mainResourceURL = nullptr;
```

Reply:
Using `nullptr` was my first choice, but for some reason at some point in development that was generating an error while `0` wasn't. (The ivar was a different type, not `WKRetainPtr<WKURLRef>`, as it is now.)